contrib/packaging/docker/Makefile

```diff
@@ -8,7 +8,7 @@ build: clean
 	cp -v ../../../Dockerfile $(BUILDDIR)
 	find $(BUILDDIR) -name ".*" -prune -exec rm -rf {} \;
 	$(MAKE) -C $(BUILDDIR) clean
-	docker build -t "cilium:cilium-ubuntu-16-04" $(BUILDDIR)
+	docker build -t "cilium:local_build" $(BUILDDIR)
```

Comment:
Let's make this an environment variable which defaults to `"latest"` instead. The relevant build script can then set that to `"local_build"`. This way the Makefile can be used outside of the automated build system as well.